portal-api/src/v1/users/sign-in-link.controller.js

```diff
@@ -1,10 +1,52 @@
+const { LOGIN_STATUSES } = require('../../constants');
+const utils = require('../../crypto/utils');
+const { updateLastLogin } = require('./controller');
+const { sanitizeUser } = require('./sanitizeUserData');
+
 class SignInLinkController {
   #signInLinkService;
 
   constructor(signInLinkService) {
     this.#signInLinkService = signInLinkService;
   }
 
+  async loginWithSignInLink(req, res) {
+    try {
+      const {
+        params: { signInToken },
+        user,
+      } = req;
+
+      const isValidSignInToken = await this.#signInLinkService.isValidSignInToken({ userId: user._id, signInToken });
+
+      if (!isValidSignInToken) {
+        const invalidSignInTokenMessage = `Invalid sign in token for user: ${user.username}`;
```

Comment:
Minor question, but is there a reason we choose to log and respond with the username instead of the id?

Reply:
Minor oversight